pkgs/development/python-modules/crcelk/default.nix

```
@@ -0,0 +1,25 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "crcelk";
+  version = "1.3";
```

```suggestion
  version = "1.3";
  format = "setuptools";

  disabled = pythonOlder "3.7";
```

I suggest to limited the support to Python release which are still maintained, especially for new packages as the chance is high that nobody will test or run it against an obsolete Python release.